### PR TITLE
Implementation of the calculation of the area of ​​the rhombus in Exercise 1

### LINKED ISSUE #4: Add exercise #1 - Rhombus area calculation
### ✨ Feature Request  

**Description:**  
As a user, I want to be able to calculate the area of a given rhombus by informing both its diagonals' values, so that I can have the right area value without needing to do complex calculates by myself.

**Motivation:**  
Since to calculate the area of a given rhombus I need to follow the formula "A = (d1 * d2) / 2", where "d1" and "d2" represents both rhombus diagonals' values, it's interesting to have a simple application which receives both diagonals' values and return the right area calculated.

**Acceptance Criteria**  
1. Given that I need to know both rhombus diagonals' values to calculate its area
When the system shows "Exercício 1 - Área do losango" page
Then the system must show two input fields to receive the rhombus diagonals' values, allowing the user to input the desired diagonals' values

2. Given that I need a trigger to calculate the area with the given rhombus diagonals' values
When the system shows "Exercício 1 - Área do losango" page with both input fields described above
Then the system must show a "Calcular" button to calculate the rhombus area with the given information from the input fields

3. Given that I want to calculate a given rhombus area by providing both its diagonals' values
When the user clicks on the "Calcular" button after filling both input fields
Then the system must show a message on screen with the related calculated area value "Área do losango: [calculated-area]"
> e.g. Given that the calculated area is '25', system must show "Área do losango: 25"

4. Given that I must have both input fields filled to be able to calculate the rhombus area
When the user clicks on the "Calcular" button
And the system identifies that there's any blank field
Then the system must show an error message on screen informing the user that both fields must be filled to proceed, preventing the user from proceeding
> e.g. If any diagonal value is blank/empty, then system must throw the error message and stop the script.

5. Given that I must have both diagonals' values greater than 0 to be able to calculate the rhombus area
When the user clicks on the "Calcular" button
And the system identifies that there's any value less than or equal to 0
Then the system must show an error message on screen with the text "Valores inválidos", preventing the user from proceeding
> e.g. If any diagonal value is less than or equal to 0, then system must throw the error message and stop the script.

6. Given that I must be able to come back to the website home page
When the system shows "Exercício 1 - Área do losango" page
Then the system must show a "Voltar" button above the exercise content that redirects the user back to the website home page when clicked

**Additional Notes:**  
- Input labels can be named as needed, there's no specific rule for that.
- Exercise page must be responsive.

### PR DESCRIPTION
**Pull request by Loogan**
✨ Pull Request Overview 
This PR adds the functionality for calculating the area of ​​a rhombus in Exercise 1 and closes #4.

📌 Why these changes? 
-These changes are necessary to implement the logic for calculating the area of ​​a rhombus based on the diagonals' values ​​provided by the user. The form now includes input fields for the diagonals and performs validation to ensure proper values ​​are entered. This improves the user experience by allowing seamless interaction and calculation within the exercise.

Feel free to provide any additional feedback or suggestions! 🙌

### CODE REVIEW

File: exercices/exercicio-1/index.html [hunk 1]
```diff
@@ -4,8 +4,23 @@
...
+  <header>
+    <div class="container">
+      <a href="/index.html">⇦ Voltar</a>
+    </div>
+  </header>
+  <main>
+    <div class="calculo container"> 
+      <h1>Calcular a área do losango</h1>
+      <input type="number" name="" id="d1" placeholder="Diagonal 1">
+      <input type="number" name="" id="d2" placeholder="Diagonal 2">
```
Comment (reenatoteixeira) [inline exercices/exercicio-1/index.html:19]: 1. Given that it's a best practice to use labels when working with inputs, we should add a `label` here for each available input.
2. Given that the `name` attributes are not being used, we can remove them to improve code readability.
3. Given that by adding labels, the placeholder becomes redundant, we can either remove or change them.
```html
      <label for="d1">Diagonal 1:</label>
      <input type="number" id="d1">
      
      <label for="d2">Diagonal 2:</label>
      <input type="number" id="d2">
```

**Review by reenatoteixeira — CHANGES_REQUESTED** [review]
There are some minor changes to the exercise 1 structure that must be applied.

**Comment by reenatoteixeira** [comment]
> ✨ Pull Request Overview
This PR adds the functionality for calculating the area of ​​a rhombus in Exercise 1 and closes #1.
> 
> 📌 Why these changes?
-These changes are necessary to implement the logic for calculating the area of ​​a rhombus based on the diagonals' values ​​provided by the user. The form now includes input fields for the diagonals and performs validation to ensure proper values ​​are entered. This improves the user experience by allowing seamless interaction and calculation within the exercise.
> 
> Feel free to provide any additional feedback or suggestions! 🙌

We're closing the wrong issue (#1) here. Please, update the issue number to the correct issue so we don't accidentally close a different issue.

CC @Loogan

File: exercices/exercicio-1/index.html [hunk 2]
```diff
@@ -4,8 +4,23 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Desafios | Exercicio - 1</title>
+  <link rel="stylesheet" href="/assets/css/exercicio1.css">
 </head>
 <body>
-  
+  <header>
+    <div class="container">
+      <a href="/index.html">⇦ Voltar</a>
```
Comment (reenatoteixeira) [inline exercices/exercicio-1/index.html:12]: This `href` attribute will cause problems when deployed. Don't fix it for now, let's leave as it is so we can open a bug-issue to fix it in the future (just to practice different issues/branches/commits).

File: exercices/exercicio-1/index.html (hunk 1 above)
Comment (Loogan) [reply exercices/exercicio-1/index.html]: new commit done the problem, please review again.

**Pull request merged**